lib/bulbs-elements/util/in-view-monitor.js

```diff
@@ -120,7 +125,7 @@ function handleDisplayMutation () {
   }
 }
 
-export default {
+const inViewMonitor = {
```

Comment:
👍 @MelizzaP and I paired on this one.

There might be a better way to manage this, but this is sort of a like garbage collection for our in-view-monitoring which is probably a good thing to do. Otherwise this loop will crash if you leave an element in a monitoring state.